timedctl/timedctl.py

```diff
@@ -25,708 +24,568 @@
-    error_handler("ERR_NOT_IMPLEMENTED")
-
-
-@edit.command("absence", aliases=["abs"])
-def edit_absence():
-    """Edit absence(s)."""
-    error_handler("ERR_NOT_IMPLEMENTED")
-
-
-@timedctl.group(cls=ClickAliasedGroup, aliases=["ac"])
-def activity():
-    """Do stuff with activities."""
-    # pylint: disable=W0107
-
-
-@activity.command("start", aliases=["add", "a"])
-@click.argument("comment")
-@click.option("--customer", default=None)
-@click.option("--project", default=None)
-@click.option("--task", default=None)
-@click.option("--show-archived", default=False, is_flag=True)
-def start_activity(comment, customer, project, task, show_archived):
-    """Start recording activity."""
-    task_id = select_task(customer, project, task, show_archived)
-    # create the activity
-    res = timed.activities.start(
-        attributes={"comment": comment},
-        relationships={"task": task_id},
-    )
-
-    if res.status_code == requests.codes["created"]:
-        msg(f"Activity {comment} started successfully.")
-        return
-    # handle exception
-    error_handler("ERR_ACTIVITY_START_FAILED")
-
-
-@activity.command("stop", aliases=["end", "finish"])
-def stop_activity():
-    """Stop current activity."""
-    if not timed.activities.current:
-        error_handler("ERR_NO_CURRENT_ACTIVITY")
-    else:
-        timed.activities.stop()
-    msg("Activity stopped successfully.")
-
-
-@activity.command("show", aliases=["s", "get", "info"])
-@click.option("--short", default=False, is_flag=True)
-def show_activity(short):
-    """Show current activity."""
-    current_activity = timed.activities.get(
-        filters={"active": True},
-        include="task,task.project,task.project.customer",
-    )
-    if current_activity:
-        activity_obj = current_activity[0]
-        comment = " > " + activity_obj["attributes"]["comment"] if not short else ""
-        start = activity_obj["attributes"]["from-time"].strftime("%H:%M:%S")
-        msg(
-            f"Current activity: {format_activity(activity_obj)}{comment} (Since "
-            + f"{start})",
+        error_handler("ERR_REPORT_CREATION_FAILED")
+
+    def edit_report(self, date):
+        """Edit report(s)."""
+        report = self.select_report(date)
+
+        msg("Comment", True)
+        comment = click.prompt("", default=report[1].strip())
+        duration = time_picker(default=report[2])
+        res = pyfzf.FzfPrompt().prompt(["No", "Yes"], "--prompt 'Are you sure?'")
+        if res == ["Yes"]:
+            res = self.timed.reports.patch(
+                report[-1],
+                {"comment": comment, "duration": duration},
+                {"task": report[-2]},
+            )
+            if res.status_code == requests.codes["ok"]:
+                msg("Report updated successfully")
+                return
+            # handle exception
+            error_handler("ERR_REPORT_UPDATE_FAILED")
+        else:
+            error_handler("ERR_REPORT_UPDATE_ABORTED")
```

Comment:
The `else` is not necessary
```suggestion
        error_handler("ERR_REPORT_UPDATE_ABORTED")
```